nengo_gui/static/side_menu.js

```diff
@@ -0,0 +1,189 @@
+
+/**
+ * Menu for the side of the GUI
+ *
+ * SideMenu constructor is written into HTML file by python and called
```

Comment:
Once this component gets merged, it should be added to the overview documentation. Do we want to log issues for that documentation on GitHub or just in Google Drive? Does anyone know what I'm talking about when I say "overview documentation"?